From 040687f1f99f9354d978700a4e3bdc8d4b1db88a Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 28 Aug 2018 10:32:25 +0800
Subject: [PATCH] arm: dts: rockchip: rk3288: Assign npll to 1250MHz

In order to support 420MHz for gpu and 125MHz, 50MHz for gmac.

Change-Id: I2b0e3edbf08850555c5bd4bc1d063c8923d54bda
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm/boot/dts/rk3288.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index 8e51132ef7e4..8abc4cee3d16 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -1051,7 +1051,7 @@
                                   <&cru ACLK_PERI>, <&cru HCLK_PERI>,
                                   <&cru PCLK_PERI>;
 		assigned-clock-rates = <594000000>,
-				       <500000000>, <300000000>,
+				       <1250000000>, <300000000>,
 				       <150000000>, <75000000>,
 				       <300000000>, <150000000>,
 				       <75000000>;
-- 
2.35.3

